fabischapeli/fast-food-fast-v1
# #160232241 this pr updates the meal status in existing meal

What does this PR do?

This pr allows the admin to update the meal in the application

Description of Task to be completed?

Changing the state of an existing meal for example, if the price was Ksh. 500, you change it to Ksh. 240 

How should this be manually tested?

Fork this branch to your github, clone it to your local .
Go to chrome, more tools, extensions and add postman
once in postman, make sure the server is running and include the relevent routes and methods you want to test then click send. 
You should get the json format of the updated meal as a return else, there is a bug in your code or the server is not running

Any background context you want to provide?

The server code has been written using flask mini framework

What are the relevant pivotal tracker stories?

#160232241
